public/src/main/java/com/revenuecat/purchases/errors.kt

```diff
@@ -32,7 +32,6 @@ enum class PurchasesErrorCode(val description: String) {
     NetworkError("Error performing request."),
     InvalidCredentialsError("There was a credentials issue. Check the underlying error for more details."),
     UnexpectedBackendResponseError("Received malformed response from the backend."),
-    ReceiptInUseByOtherSubscriberError("The receipt is in use by other subscriber."),
```

Comment:
this one was found to be redundant, right?

Reply:
@aboedo yup! We originally mapped this to error code 7102 in Android, and Android did not use `RCReceiptAlreadyInUseError`. While iOS uses `RCReceiptAlreadyInUseError`, but not `ReceiptInUseByOtherSubscriberError`.  This PR is to make it consistent between iOS/Android.